packages/auditd_manager/data_stream/auditd/fields/fields.yml

```
@@ -623,9 +623,11 @@
 - name: auditd.data.perm_mask
   description: file permission mask that triggered a watch event
   type: keyword
-- name: auditd.data.a0-N
-  description: the arguments to a syscall
-  type: keyword
+# this mapping does not generate a dynamic template, and the expected fields do not match
+# should it be kept for documentation purposes?
+# - name: auditd.data.a0-N
+#   description: the arguments to a syscall
+#   type: keyword
```

What should it be done for this field definition ? Just remove it?

---

Would it work with a `auditd.data.a*` definition, specially to keep the documentation?
```suggestion
 - name: auditd.data.a*
   description: the arguments to a syscall
   type: keyword
```
Though this would match also things that are not arguments.

So maybe this can be removed, yes.